### PR TITLE
testing the styling of pagination per their documentation

### PR DESCRIPTION
**Pull request by xscottxbrownx**
### This PR:
1. Styles the `prev` and `next` buttons as simple chevrons.
2. Styles all the pagination buttons using our color theme.

---

### The two files this PR effects:
`MessageFolder.jsx`
`MessageStyles.jsx`

---

I left the styles that were being used in `MessageStyles` commented out, with new styling underneath it.

Screenshot 2023-07-13 at 5 56 15 PM

### CODE REVIEW

File: src/components/Messages/MessageStyles.js [hunk 1]
```diff
@@ -31,30 +31,62 @@ export const StyledButton2 = styled('button')({
...
   }
 
-  a:active {
-    background-color: #555;
-    color: #fff;
+  .pagination .page:hover {
+    background-color: #004d3e;
+  }
+
+  .active-page {
```
Comment (xscottxbrownx) [inline src/components/Messages/MessageStyles.js]: having issues with this

File: src/components/Messages/MessageFolder.jsx [hunk 2]
```diff
@@ -85,12 +85,17 @@ const MessageFolder = ({ folderType, handleRefresh, loadMessages, messageList })
...
             pageRangeDisplayed={5}
             pageCount={pageCount === 0 ? 1 : pageCount}
-            previousLabel="< previous"
+            previousLabel="<"
             renderOnZeroPageCount={null}
+            className="pagination"
+            previousLinkClassName="page"
+            nextLinkClassName="page"
+            pageLinkClassName="page"
+            activeClassName="active-page"
```
Comment (xscottxbrownx) [inline src/components/Messages/MessageFolder.jsx]: having issues with this

**Comment by xscottxbrownx** [comment]
Spent some time tonight and figured it out...

### Now need votes on Yellow or Red `prev` and `next` pagination buttons.


https://github.com/codeforpdx/PASS/assets/71395931/392bc897-76cf-459a-bac7-ccbdeff28ffe


https://github.com/codeforpdx/PASS/assets/71395931/e8260830-48c2-4d06-9a5e-11a9e8cc8dae

**Comment by leekahung** [comment]
> Spent some time tonight and figured it out...
> 
> ### Now need votes on Yellow or Red `prev` and `next` pagination buttons.
> 
> 
> https://github.com/codeforpdx/PASS/assets/71395931/392bc897-76cf-459a-bac7-ccbdeff28ffe
> 
> 
> https://github.com/codeforpdx/PASS/assets/71395931/e8260830-48c2-4d06-9a5e-11a9e8cc8dae

I think red is better compared to yellow in this case, it pops out more. A bit unsure about the yellow text for the selected page number though. Not as good as I initially thought when we discussed about the pagination.

**Comment by leekahung** [comment]
For the `prev` and `next` buttons, I see that it utilizes the text value as the "icons" for those buttons. Is there any way to change them to MUI icons instead, or it's not really possible with this library?

Curious about this since I just learned there's a native paginate component in MUI.

**Comment by xscottxbrownx** [comment]
> For the `prev` and `next` buttons, I see that it utilizes the text value as the "icons" for those buttons. Is there any way to change them to MUI icons instead, or it's not really possible with this library?
> 
> Curious about this since I just learned there's a native paginate component in MUI.

### You can set them as icons...
Screenshot 2023-07-14 at 7 25 14 AM

Screenshot 2023-07-14 at 7 41 48 AM

**Review by timbot1789 — COMMENTED** [review]
I like the red more too. IMO it's good to have a few more colors in the display.

File: src/components/Messages/MessageFolder.jsx [hunk 3]
```diff
@@ -4,9 +4,12 @@ import ReactPaginate from 'react-paginate';
 // Styling Imports
```
Comment (leekahung) [inline src/components/Messages/MessageFolder.jsx]: Think we can just group up all the MUI Imports together and remove `Styling Imports`

**Review by leekahung — APPROVED** [review]
I'm pretty much in favor of approving this. Code looks fine and the interactions with the paginations seems to be working well.

The comment for the Styling Imports is minor and doesn't need to be included.

**Comment by andycwilliams** [comment]
MUI does have a component specifically for pagination:

https://mui.com/material-ui/react-pagination/

Although maybe we can just leave fully switching to that for a future update.

**Review by timbot1789 — APPROVED** [review]

File: src/components/Messages/MessageFolder.jsx (hunk 3 above)
Comment (xscottxbrownx) [reply src/components/Messages/MessageFolder.jsx]: Changed!

Sorry, I missed this comment earlier.

**Comment by xscottxbrownx** [comment]
> I'm pretty much in favor of approving this. Code looks fine and the interactions with the paginations seems to be working well.
> 
> The comment for the Styling Imports is minor and doesn't need to be included.

I just grouped all the MUI imports together. Sorry for delay.

**Pull request merged**